Skype/WebSDK/docs/PTChat.md

```diff
@@ -1,4 +1,7 @@
-
+---
+title: Chat
```

Comment:
same

Reply:
I was following the instructions on the Procedures tab of the website that say to have the title match the H1.